h/static/scripts/annotation-ui.js

```diff
@@ -96,8 +100,23 @@ function annotationsReducer(state, action) {
   }
 }
 
+function sortReducer(state, action) {
+  switch (action.type) {
+    case types.SORT_BY:
+      return Object.assign({}, state, {name: action.mode});
+    case types.SET_SORT_OPTIONS:
+      return Object.assign({}, state, {
+        name: action.mode,
```

Comment:
It would be nice if we could take this opportunity to unify the naming here. We're calling it "name" in the object, but "mode" in the action. And "name" is a pretty odd choice, IMO. Perhaps refer to it as the sort "key" everywhere, and rename "options" to "availableKeys"?